api/README.md

```diff
@@ -790,16 +790,26 @@ __`product`__ : browser[version[os[version]]]. e.g. `chrome-63.0-linux`
 [
    {
       "test":"/IndexedDB/bindings-inject-key.html",
+      "subtest":"subtestFoo",
       "urls":[
          "bugs.chromium.org/p/chromium/issues/detail?id=934844",
          "bugs.webkit.org/show_bug.cgi?id=167052"
+      ],
+      "status":[
+         "FAIL",
+         "FAIL"
       ]
    },
```

Comment:
This format is now quite different than the format of the META.yml files (which have a single URL, and multiple results combinations to which it applies). Should we be revising the format of `/api/metadata` in a similar way to match?

Reply:
I intended to format /api/metadata response similar to [/api/search,](https://github.com/web-platform-tests/wpt.fyi/blob/44d1b0905d9b194779ac4cca03bbe6c6ac256b54/api/README.md#apisearch). I think the current format (each combinations of test and subtest has a URL) is easy to use if users are searching for test names instead of URLs.